source/common/protobuf/utility.cc

```diff
@@ -48,9 +48,16 @@ ProtoValidationException::ProtoValidationException(const std::string& validation
   ENVOY_LOG_MISC(debug, "Proto validation error; throwing {}", what());
 }
 
+bool MessageUtil::allow_unknown_fields = false;
```

Comment:
Just for clarity, can we make this an enum with 2 states? Then the function can take the enum value like I mentioned before (and CLI can convert to enum).

Reply:
Done, let me know if I need to change the names of anything to conform with the style.